File: javalibrary/src/main/java/com/bluthlee/javalibrary/data/DataBinder.java
package com.bluthlee.javalibrary.data;

import android.view.View;

import com.bluthlee.javalibrary.AudioRecorder;

/**
 * Created by LC on 2017/8/31.
 */
public class DataBinder {

    public static final int STYLE_DRAG = 0;
    public static final int STYLE_FLOAT = 1;
    public static final int STYLE_THREE = 2;

    public AudioRecorder.OnFinishListener onFinishListener = null;

    public int style = STYLE_DRAG;

    public View buttonView = null;

    public String fileName = "";

    public String dirPath = "";

}

File: kotlinlibrary/src/main/java/com/bluthlee/library/data/DataBinder.kt
package com.bluthlee.library.data

import android.view.View
import com.bluthlee.library.AudioRecorder

/**
 * Created by LC on 2017/8/24.
 */
internal class DataBinder {

    companion object {
        const val STYLE_DRAG = 0
        const val STYLE_FLOAT = 1
        const val STYLE_THREE = 2
    }

    var onFinishListener: AudioRecorder.OnFinishListener? = null

    var style: Int = STYLE_DRAG

    var buttonView: View? = null

    var fileName = ""

    var dirPath = ""

}